genestack_client/genestack_metainfo.py

```diff
@@ -31,6 +31,7 @@ class Metainfo(dict):
     CONTACT_PERSON = 'genestack:contactPerson'
     EXTERNAL_LINK = 'genestack:links'
     CREATION_DATE = 'genestack:dateCreated'
+    PARENT_DICTIONARY = 'genestack.dictionary:parent'
```

Comment:
I'm afraid that this key must not be placed here... Need to consult with @genestack-solomatin though.